spec/support/file_fixture_support.rb

```diff
@@ -1,4 +1,12 @@
 module FileFixtureSupport
+  def load_yaml(fixture_name)
+    if RUBY_VERSION >= '3.0'
```

Comment:
The default psych gem version of ruby 3.0.x is `3.3.2` ...? 🤔

I think `YAML.respond_to?(:unsafe_load_file)` or `YAML::VERSION >= "4"` is better.
(It's OK because it only used in the test.)

Reply:
Oops, my code was silly, it should be `3.1` instead of `3.0`, and as you say it's smarter to use `respond_to?`.
I created a follow-up pull request https://github.com/x-motemen/git-pr-release/pull/74. 🙏